packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -105,7 +123,7 @@ All done! To run your application:
     String projectIdentifier = _createProjectIdentifier(path.basename(dirPath));
     String relativeFlutterPackagesDirectory = path.relative(flutterPackagesDirectory, from: dirPath);
 
-    printStatus('Creating ${path.basename(projectName)}...');
+    printStatus('Creating project ${path.basename(projectName)}...');
```

Comment:
We use the word project here: https://flutter.io/getting-started/; I'll stick with `project` for consistency.